vital/types/metadata_traits.h

```diff
@@ -40,7 +41,8 @@ struct vital_meta_trait_base
   virtual bool is_signed() const = 0;
   virtual bool is_floating_point() const = 0;
   virtual vital_metadata_tag tag() const = 0;
-  virtual std::unique_ptr<metadata_item> create_metadata_item( const kwiver::vital::any& data ) const = 0;
+  virtual std::unique_ptr<metadata_item> create_metadata_item(
```

Comment:
Seems a bit odd to break here rather than
```
virtual std::unique_ptr< metadata_item >
create_metadata_item( any const& data ) const = 0;
```
given that we always break after return type in other contexts. Not too important, though.